go/libraries/doltcore/sqle/dprocedures/init.go

```diff
@@ -30,6 +30,7 @@ var DoltProcedures = []sql.ExternalStoredProcedureDetails{
 	{Name: "dolt_commit", Schema: stringSchema("hash"), Function: doltCommit},
 	{Name: "dolt_commit_hash_out", Schema: stringSchema("hash"), Function: doltCommitHashOut},
 	{Name: "dolt_conflicts_resolve", Schema: int64Schema("status"), Function: doltConflictsResolve},
+	{Name: "dolt_count_commits", Schema: int64Schema("ahead", "behind"), Function: doltCountCommits},
```

Comment:
Is this a public procedure? Do we need to document this somewhere for user consumption?

Reply:
Yup, this is a public procedure. We'll need to add documentation for it to https://github.com/dolthub/docs
I'll be opening a PR for docs after this PR is merged.